### PR TITLE
edit UI rich text for design

### PR DESCRIPTION
**Pull request by ManhNTX**
Screen Shot 2022-07-26 at 06 55 57

### CODE REVIEW

File: lib/features/composer/presentation/composer_view.dart [hunk 1]
```diff
@@ -463,83 +445,118 @@ class ComposerView extends GetWidget<ComposerController> with AppLoaderMixin, Ri
...
+                  uploadAttachments,
+                  controller.expandModeAttachments.value))
+        ]);
+      }
+    });
+  }
+
+  Widget _buildInlineLoadingView() {
+    return Obx(() => controller.uploadController.uploadInlineViewState.value.fold(
+            (failure) => const SizedBox.shrink(),
```
Comment (dab246) [inline lib/features/composer/presentation/composer_view.dart:481]: Indent too much.

File: lib/features/composer/presentation/composer_view.dart [hunk 2]
```diff
@@ -463,83 +445,118 @@ class ComposerView extends GetWidget<ComposerController> with AppLoaderMixin, Ri
...
+            (success) {
```
Comment (dab246) [inline lib/features/composer/presentation/composer_view.dart:482]: same

File: lib/features/composer/presentation/mixin/rich_text_button_mixin.dart [hunk 3]
```diff
@@ -80,6 +80,34 @@ mixin RichTextButtonMixin {
...
+
+    return SvgPicture.asset(path,
+        color: newColor?.withOpacity(opacity),
+        fit: BoxFit.fill);
+  }
+
+  Widget buildIconColorBackgroundTextWithoutTooltip({
+    required IconData? iconData,
+    required Color? colorSelected,
+    String? tooltip,
```
Comment (dab246) [inline lib/features/composer/presentation/mixin/rich_text_button_mixin.dart:100]: should remove it if not use

**Comment by dab246** [comment]
Screen Shot 2022-07-26 at 09 13 26

As the demo image, I see the spacing between the buttons and the uneven height. Please check again.

**Review by hoangdat — APPROVED** [review]

**Review by dab246 — APPROVED** [review]

**Pull request merged**